modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -77,6 +77,14 @@ object TrustStorage {
 
       def getTrust: F[TrustMap] = trustStoreRef.get.map(_.trust)
 
+      def getBiasedTrustScores: F[TrustScores] = getTrust
+        .map(getBiasedTrust(_).trust)
+        .map(_.view.mapValues { trustInfo =>
+          trustInfo.predictedTrust.orElse(trustInfo.trustLabel)
+        }.toMap)
+        .map(_.collect { case (peerId, Some(trustScore)) => peerId -> trustScore })
+        .map(scores => TrustScores(scores))
```

Comment:
`.map(TrustScores)` or `.map(TrustScores(_))` or `.map(TrustScores.apply)`

Reply:
Changed to `.map(TrustScores(_))`.